devfile.yaml

```diff
@@ -81,7 +90,7 @@ commands:
       component: tools
       workingDir: ${PROJECTS_ROOT}/java-spring-petclinic
       commandLine: |
-        java -jar -Dspring-boot.run.profiles=mysql \
+        java -jar -Dspring.profiles.active=mysql \
```

Comment:
run-with-mysql ---> run-with-mariadb ?

Reply:
Mariadb is a fork of mysql. I think we can keep the existing name.